gdk/src/auto/device_pad.rs

```diff
@@ -51,22 +39,16 @@ impl<O: IsA<DevicePad>> DevicePadExt for O {
         }
     }
 
-    #[cfg(any(feature = "v3_22", feature = "dox"))]
-    #[cfg_attr(feature = "dox", doc(cfg(feature = "v3_22")))]
     fn get_group_n_modes(&self, group_idx: i32) -> i32 {
         unsafe { ffi::gdk_device_pad_get_group_n_modes(self.as_ref().to_glib_none().0, group_idx) }
     }
 
-    #[cfg(any(feature = "v3_22", feature = "dox"))]
-    #[cfg_attr(feature = "dox", doc(cfg(feature = "v3_22")))]
     fn get_n_features(&self, feature: DevicePadFeature) -> i32 {
         unsafe {
             ffi::gdk_device_pad_get_n_features(self.as_ref().to_glib_none().0, feature.to_glib())
         }
     }
 
-    #[cfg(any(feature = "v3_22", feature = "dox"))]
-    #[cfg_attr(feature = "dox", doc(cfg(feature = "v3_22")))]
```

Comment:
This shouldn't have been removed based on the documentation: https://developer.gnome.org/gdk3/stable/GdkDevicePad.html#gdk-device-pad-get-n-groups